Sephora-M/assignment3/LetterGridTest.java

```diff
@@ -0,0 +1,50 @@
+package assignment3;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Note: should improve test by using setup and by testing edge cases...
+ * @author Sephora-M
+ *
+ */
+class LetterGridTest {
+	
+	@Test
```

Comment:
I love how you tested the `isWord` method! :100: 
How about also testing the `isPrefix` method?